### PR TITLE
M5 stickC fix

### PR DESCRIPTION
**Pull request by malw4**

### CODE REVIEW

File: docs/Anjay_integrations/ESP32_integration/M5Stick.md [hunk 1]
```diff
@@ -13,7 +13,27 @@ Integrate your ESP32-based device to manage it via Coiote DM.
...
+
+0. Log in to Coiote DM and from the left side menu, select **Device Inventory**.
+0. In **Device Inventory**, click **Add device**.
+0. Select the **Connect your LwM2M device directly via the Management server** tile.
+![Add via Mgmt](images/mgmt_tile.png "Add via Mgmt")
+0. In the **Device credentials** step:
+    - In the **Device ID** field, type the endpoint name provided in the `nvs_config.csv`, e.g. `ESP32_test`.
+    - In the **Security mode** section, select the **PSK** mode.
+    - In the **Key identity** field, type the name provided in the `nvs_config.csv`, e.g. `ESP32_test`.
+    - In the **Key identity** field, type the identity provided in the `nvs_config.csv`, e.g. `ESP32_test`.
```
Comment (JZimnol) [inline docs/Anjay_integrations/ESP32_integration/M5Stick.md:30]: Doubled the `Key identity` step. Also it is worth mentioning that `It is recommended to use endpoint name as key identity.`

File: docs/Anjay_integrations/ESP32_integration/M5Stick.md [hunk 2]
```diff
@@ -13,7 +13,27 @@ Integrate your ESP32-based device to manage it via Coiote DM.
...
+    - In the **Key** field, type the `psk` key provided in the `nvs_config.csv`.
```
Comment (JZimnol) [inline docs/Anjay_integrations/ESP32_integration/M5Stick.md:31]: The `nvs_config.csv` file isn't mentioned earlier. What is this file? Only later the "3." step mentions that file. I would have no idea what to do honestly.

File: docs/Anjay_integrations/ESP32_integration/M5Stick.md [hunk 3]
```diff
@@ -13,7 +13,27 @@ Integrate your ESP32-based device to manage it via Coiote DM.
...
+    ![Device credentials step](images/add_mgmt_quick.png "Device credentials step")
+0. Click the **Add device** button and **Confirm** in the confirmation pop-up.
+0. In the **Connect your device** step, the server is waiting for the board to connect. You can now start connecting the device.
+
+## Step 3: Configure the client using an NVS partition
 
 0. Create a `nvs_config.csv` file and save it in your project directory. In the file, provide your credentials in [wifi_ssid], [wifi_password], [endpoint_name], [identity], [psk], [lwm2m_server_uri] (without the `[]` brackets). Use the following snippet as a template:
```
Comment (JZimnol) [inline docs/Anjay_integrations/ESP32_integration/M5Stick.md:38]: Note: that file can also be downloaded from [https://github.com/AVSystem/Anjay-esp32-client/releases](https://github.com/AVSystem/Anjay-esp32-client/releases)

File: docs/Anjay_integrations/ESP32_integration/M5Stick.md [hunk 4]
```diff
@@ -42,52 +62,66 @@ Integrate your ESP32-based device to manage it via Coiote DM.
...
-![Client configuration](images/nvs_config.png "Client configuration"){: style="float: left;margin-right: 1177px;margin-top: 17px;"}
+    ![Client configuration](images/nvs_config.png "Client configuration"){: style="float: left;margin-right: 1177px;margin-top: 17px;margin-bottom: 30px;"}
 
-## Step 3: Add device to Coiote DM
-
-To connect your M5StickC to the Coiote IoT Device Management LwM2M Server, use your access to a Coiote DM installation, or register at https://eu.iot.avsystem.cloud/ to get access.
-
-To connect the board:
+    !!! tip
+        Before flashing the device you should erase the flash first to make sure you have correct settings.
```
Comment (JZimnol) [inline docs/Anjay_integrations/ESP32_integration/M5Stick.md:79]: I would move this  to the "4." step. Also, how to erase the flash?

File: docs/Anjay_integrations/ESP32_integration/M5Stick.md (hunk 2 above)
Comment (malw4) [reply docs/Anjay_integrations/ESP32_integration/M5Stick.md:31]: nvs_config.csv has his own step.  I think step 2 "Configure the client using an NVS partition" describes everything step by step.

**Pull request closed without merging**